src/azure-cli/azure/cli/command_modules/sqlvm/_params.py

```diff
@@ -205,9 +216,6 @@ def load_arguments(self, _):
                    help='SQL Server management type. Updates from LightWeight to Full.',
                    options_list=['--sql-mgmt-type'],
                    arg_type=get_enum_type(['Full']))
-        c.argument('prompt',
-                   options_list=['--yes', '-y'],
-                   help="Do not prompt for confirmation. Requires --sql-mgmt-type.")
```

Comment:
Can you remove the prompt logic in `custom.py` but keep the argument until next breaking change window to avoid break customers' scripts suddenly? You can mark this argument as deprecated by adding `deprecate_info=c.deprecate()`.

Reply:
Added the argument back and added the deprecate_info as requested.